tools/.golint_failures

```diff
@@ -1 +1,2 @@
 # Apart from this line, only trailing comments are supported
+pkg/controller/nodeipam/config/v1alpha1
```

Comment:
Are there a lot of lint failures?

Reply:
Some function names not passing the check in auto generate files